packages/icons/src/NFTs.tsx

```diff
@@ -19,5 +19,5 @@ export function Reload(props: SvgIconProps) {
 }
 
 export function Copy(props: SvgIconProps) {
-  return <CopyIcon component={ReloadIcon} viewBox="0 0 22 22" {...props} />;
+  return <SvgIcon component={CopyIcon} viewBox="0 0 22 22" {...props} />;
```

Comment:
Looks like the prior code was a hacky change to update all of the CopyToClipboard icons.

With this change, all of the Copy icons are incorrect:

![image](https://user-images.githubusercontent.com/339312/214651476-c6ba9257-3fd6-490d-8142-9ee488d882fa.png)